defaults/main.yml

```diff
@@ -84,4 +84,5 @@ beegfs_client: []
 #   port: 8004 # If there are multiple clients, use different port for each client.
 #   mgmt_host: # Use this to override the default beegfs_mgmt_host in cases there are multiple clients.
 
+beegfs_package: present # choose `latest` to upgrade
```

Comment:
As per our discussion, this variable could be more clearly named.
We've got two factors at play here - the major version (7.0, 7.1, etc), for which there may be version-dependent actions to consider (eg, the RDMA-enabling library).  This might also set a specific repo from which to pull the packages.
The other factor is whether to update a system to the latest release of that major version.
One possibility would be `beegfs_version`, which could default to `latest` but (in future) support specific numbered releases.  Then another boolean flag, `beegfs_update`, which would update packages to latest available from the current repo.
My main concern is that these additional controls are introduced with default values that don't break existing users.  I think the default values should be `latest` and `false`.

I suggest we focus on `beegfs_update` in this review and create a separate RFE for `beegfs_version`, which will need more thought.

Reply:
I have introduced a beegfs_update boolean flag that specifies what should be the content of beegfs_package.